File: pinyougou-parent/pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbBrand;
import entity.PageResult;

import java.util.List;

public interface BrandService {
    public List<TbBrand> findAll();
    public PageResult findPage(int pageNo,int pageSize);
    public  void add(TbBrand tbBrand);
    public  TbBrand findOne(long id);
    public  void update(TbBrand tbBrand);
    public  void delete(long [] ids);
    public  PageResult findPage(TbBrand tbBrand,int pageNo,int pageSize);
}

File: pinyougou-parent/pinyougou-manager-web/src/main/webapp/js/service/brandService.js
//定义服务
app.service('brandService', function ($http) {
    this.add = function (entity) {
        return $http.post("../brand/add.do", entity);
    }
    this.findPage = function (page, rows) {
        return $http.get('../brand/findPage.do?page=' + page + "&rows=" + rows);
    }
    this.findOne = function (id) {
        return $http.get('../brand/findOne.do?id=' + id);
    }
    this.update = function (entity) {
        return $http.post('../brand/update.do', entity);
    }
//            this.save=function (methodName,entity) {
//                return  $http.post('../brand/' + methodName,entity);
//            }
    this.dele = function (selectIds) {
        return $http.get('../brand/delete.do?ids=' + selectIds);
    }
    this.search = function (page, rows, searchEntity) {

        return $http.post('../brand/search.do?page=' + page + "&&rows=" + rows, searchEntity);
    }
    //下拉列表数据
    this.selectOptionList=function(){
        return $http.get('../brand/selectOptionList.do');
    }
})